netbox_agent/logging.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+
+def init_log():
+    logger = logging.getLogger('netbox_agent')
```

Comment:
If you don't plan to have much things use netbox_agent as a library, I'd try to modify the root logger directly so that you can use the root functions of `logging` and not care to remember where is the logger you should import, i.e. do `import logging; logging.debug(...)` rather than `from netbox_agent.logging import logger; logger.debug(...)`.

Reply:
done :) thanks!